components/form/rangeDatepicker/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@schibstedspain/sui-form-range-datepicker",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "rm -Rf ./lib && mkdir -p ./lib && npm run build:js && npm run build:styles",
+    "build:js": "../../../node_modules/.bin/babel --presets sui ./src --out-dir ./lib",
+    "build:styles": "../../../node_modules/.bin/cpx './src/**/*.scss' ./lib"
+  },
+  "dependencies": {
+    "@schibstedspain/sui-component-dependencies": "latest",
+    "@schibstedspain/sui-svgiconset": "1",
+    "@schibstedspain/sui-button-basic": "1.1.0",
+    "react-datepicker": "0.52.0",
+    "moment": "2"
```

Comment:
That is a BIG dependencie. Are you sure to use this ?!

Reply:
Yru're right.
But, the used package react-datepicker expects a moment instance as props. And it's already has moment as a dependency...
https://github.com/Hacker0x01/react-datepicker/blob/master/package.json#L96
To avoid moment, we would need to get rid of react-datepicker and find another one or develop the logic from scratch.

Reply:
Maybe we have two solutions:

1.- https://github.com/date-fns/date-fns/issues/275#issuecomment-264934189
2.- https://github.com/jmblog/how-to-optimize-momentjs-with-webpack

Reply:
We will try to delay the load of this component as much as in possible.